data-prepper-core/src/test/java/org/opensearch/dataprepper/event/DefaultLogEventBuilderFactoryTests.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.event;
+
+import org.opensearch.dataprepper.model.event.EventMetadata;
+import org.opensearch.dataprepper.model.log.JacksonLog;
+import org.opensearch.dataprepper.model.event.LogEventBuilder;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.CoreMatchers.not;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.apache.commons.lang3.RandomStringUtils;
+
+import java.util.Map;
+import java.util.Collections;
+
+class DefaultLogEventBuilderFactoryTests {
+    private DefaultLogEventBuilderFactory defaultLogEventBuilderFactory;
+
+    public DefaultLogEventBuilderFactory createObjectUnderTest() {
+        return new DefaultLogEventBuilderFactory();
+    }
+
+    @BeforeEach
+    public void setup() {
+        defaultLogEventBuilderFactory = createObjectUnderTest();
+    }
+
+    @Test
+    public void testBasic() {
+        assertThat(defaultLogEventBuilderFactory.getEventClass(), equalTo(LogEventBuilder.class));
```

Comment:
This assertion can be its own test case. You can remove it and put it in a test like: `getEventClass_returns_LogEventBuilder()`.